### DISCUSSION TITLE
Tags discussion

### DISCUSSION THREAD

**Discussion by ShivamMenda** [AOSSIE-Org/Resonate#202] [Ideas]
Starting a discussion to ponder about the tags while creating the room. Currently we are storing it in a list and to enable features with the tag we need to pass it as a list and in the same order. Can we change the way the tags are stored in the database so that we can work on features such as filtering etc?

**Comment by Aarush-Acharya** [comment]
Hey @ShivamMenda so we are to store it in a Map kind of form right for faster filtering if I am not wrong
